src/kili/use_cases/issue/__init__.py

```diff
@@ -0,0 +1,34 @@
+"""Issue Service."""
+
+from typing import List
+
+from kili.entrypoints.mutations.issue.helpers import get_labels_asset_ids_map
+from kili.gateways.kili_api_gateway import KiliAPIGateway
+from kili.gateways.kili_api_gateway.issue.types import IssueToCreateKiliAPIGatewayInput
+from kili.use_cases.issue.types import IssueToCreateUseCaseInput
+
+
+class IssueUseCases:
+    """Issue Service."""
+
+    def __init__(self, kili_api_gateway: KiliAPIGateway):
+        self._kili_api_gateway = kili_api_gateway
+
+    def create_issues(self, project_id, issues: List[IssueToCreateUseCaseInput]):
+        """Create issues with issue type."""
+        label_id_array = [issue.label_id for issue in issues]
+        label_asset_ids_map = get_labels_asset_ids_map(
+            self._kili_api_gateway, project_id, label_id_array
+        )  # should be done in the backend
```

Comment:
you can prefix comments like this with tags like `TODO`, so that they can be detected by vscode and we can find them easily

Reply:
yeah I didn't add it because it raised a pylint error. I added it to the exeptions i pylintrc